pull_request_template.md

```diff
@@ -24,7 +24,6 @@
 # Checklist:
 
 -   [ ] Able to run pr locally
--   [ ] Lighthouse performed (Accessibility 90+ minimum)
```

Comment:
how comes we have removed this in the template?

Reply:
Because we no longer value what it tells us (DAC audit stuff).